src/context/ProvideAuth.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { createContext, useState } from 'react';
+
+export const authContext = createContext();
+
+export function ProvideAuth({ children }) {
+  const [user, setUser] = useState(null);
+
+  const login = (username, password) => {
+    const loginSuccess =
+      username === process.env.AUTH_USERNAME &&
```

Comment:
It looks like you have these entered in your `.yml` file as `AUTH_EMAIL` and `AUTH_PASSWORD`, so you'll need to update them to be the same in order for this to work successfully.